samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -258,21 +420,51 @@ int main(void)
 	}
 	printk("Bluetooth initialized\n");
 
-	for (size_t i = 0U; i < ARRAY_SIZE(mock_data); i++) {
+	for (size_t i = 0U; i < ARRAY_SIZE(send_pcm_data); i++) {
 		/* Initialize mock data */
-		mock_data[i] = i;
+		send_pcm_data[i] = i;
 	}
 
 #if defined(CONFIG_LIBLC3)
-	init_lc3();
+#if defined(CONFIG_USB_DEVICE_AUDIO)
+	const struct device *hs_dev;
+
+	hs_dev = DEVICE_DT_GET(DT_NODELABEL(hs_0));
+
+	if (!device_is_ready(hs_dev)) {
+		printk("Device USB Headset is not ready");
+		return 0;
+	}
+
+	printk("Found USB Headset Device");
```

Comment:
Can you elaborate on why we are looking for a headset device? Aren't we specifically _not_ a headset here?

Reply:
Probably because it uses headset USB Audio `usb-audio-hs` devicetree binding. If the application is only from USB to Bluetooth, then it should probably switch to `usb-audio-hp`.

Reply:
Using the provided devicetree definition:

https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi#L289-L303

Reply:
> If the application is only from USB to Bluetooth, then it should probably switch to usb-audio-hp.

Is it only 1-way audio, but it doesn't look like it's compatible with `usb-audio-hp`?

Reply:
as there is already a definition in the devicetree for nrf5340 audio dk, I'd prefer to use that and not introduce a potential conflict.